paddle/operators/compare_op.cu

```diff
@@ -15,4 +15,10 @@
 #include "paddle/operators/compare_op.h"
 
 REGISTER_LOGICAL_KERNEL(less_than, GPU, paddle::operators::LessThanFunctor);
+REGISTER_LOGICAL_KERNEL(less_equal, GPU,
+                        paddle::operators::LessThanEqualFunctor);
```

Comment:
should these functors be changed too?

Reply:
Yes. Thank you for pointing that out. Changed it in the latest commit.